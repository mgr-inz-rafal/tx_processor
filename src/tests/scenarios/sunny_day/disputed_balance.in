type,client,tx,amount
deposit,1,1,100.0
deposit,1,2,50.0
dispute,1,1,
deposit,1,3,1.0
withdrawal,1,4,0.5

