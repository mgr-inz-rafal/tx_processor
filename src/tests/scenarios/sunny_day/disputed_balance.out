client,available,held,total,locked
1,50.5,100,150.5,false
